protocol/protobuf/pairing.proto

```diff
@@ -137,6 +137,7 @@ message SyncInstallationCommunity {
   bool spectated = 11;
   bytes encryption_keys = 12;
   SyncCommunityControlNode control_node = 13;
+  int64 last_opened_at = 14;
```

Comment:
hmm. I don't see where so we set this field in the sync message 🤔

Reply:
Would [this](https://github.com/status-im/status-go/pull/4514/commits/7276e361fcbeb5605afc6b6fc869a647307ddf8a#diff-12e1567f20b885c074b2a1213664d43a77fbc87c34b5dc42cdb6d4526a86460dR3026) work?